src/Upgrader/Business/Upgrader/Upgrader.php

```diff
@@ -7,13 +7,20 @@
 
 namespace Upgrader\Business\Upgrader;
 
+use Upgrader\Business\Exception\UpgraderFlowException;
 use Upgrader\Business\Upgrader\Request\UpgraderRequest;
 use Upgrader\Business\Upgrader\Response\Collection\UpgraderResponseCollection;
+use Upgrader\Business\Upgrader\Response\UpgraderResponse;
 use Upgrader\Business\Upgrader\Strategy\UpdateStrategyGeneratorInterface;
 use Upgrader\Business\VersionControlSystem\VcsInterface;
 
 class Upgrader implements UpgraderInterface
 {
+    /**
+     * @var string
+     */
+    protected const NOTHING_TO_UPDATE_OUTPUT_MESSAGE = 'The branch is up to date. No further action is required.';
```

Comment:
This has no meaning as const and should rather stay inline IMO.

Reply:
Deleted